src/js/containers/bulkDownload/BulkDownloadPageContainer.jsx

```diff
@@ -173,11 +173,11 @@ export class BulkDownloadPageContainer extends React.Component {
             params.filters.budget_subfunction = formState.budgetSubfunction.code;
         }
 
-        if (formState.period !== '') {
-            params.filters.period = formState.period;
+        if (formState.period !== 0 && !formState.period) {
```

Comment:
```suggestion
        if (!formState.period) {
```
Could be simplified since zero is falsey